watchers/target_pod_test.go

```diff
@@ -0,0 +1,77 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package watchers_test
+
+import (
+	chaosv1beta1 "github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/log"
+	"github.com/DataDog/chaos-controller/metrics/noop"
+	"github.com/DataDog/chaos-controller/watchers"
+	"github.com/DataDog/chaos-controller/watchers/mocks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var _ = Describe("target pod", func() {
+	Describe("Handler", func() {
+		//	@ToDo
+	})
+
+	Describe("NewDisruptionTargetWatcher", func() {
+		const watcherName = "name"
+
+		var (
+			disruption chaosv1beta1.Disruption
+			err        error
+			watcher    watchers.Watcher
+		)
+
+		JustBeforeEach(func() {
+			// Arrange
+			var logger *zap.SugaredLogger
+			logger, err = log.NewZapLogger()
```

Comment:
FYI if you need to "test" logs you can use this:
* https://github.com/DataDog/chaos-controller/blob/8d4a392eb786c80b6dc8669f388b514bd08fce45/injector/python_test.go#L117

You may also want to use instead this:
* https://pkg.go.dev/go.uber.org/zap/zaptest

That creates a logger for testing.